starter-code/models/place.js

```diff
@@ -0,0 +1,31 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+const placeSchema = new Schema({
+  name: String,
+  type: {type: String, enum: ['coffee shop, bookstore']},
```

Comment:
there was a mistake in the enum son mongodb wasn't able to save the new places` type: {type: String, enum: ["coffee shop", "bookstore"]}`